### DISCUSSION TITLE
ESP01S HC-SR501, AM2302, PN532, BME280

### DISCUSSION THREAD

**Discussion by MrVulgaire** [arendst/Tasmota#10294] [Show and tell]
Hello,

I share here my small production for Tasmota based on ESP01S

Powered AC 230V 
(AC-DC 5V 2A Mean Well IRM-10-05)
(DC-DC 3.3V AMS1117)

I can connect :

PIR HC-SR501 & temperature sensor (BME280 or AM2302)

Or 

PN532 RFID reader

Best 

![01](https://user-images.githubusercontent.com/23582534/103243469-3d581880-495a-11eb-991e-f6b86a9a8717.jpg)
![03](https://user-images.githubusercontent.com/23582534/103243471-40eb9f80-495a-11eb-974f-0d8d14ffdf2c.jpg)
![05](https://user-images.githubusercontent.com/23582534/103243473-41843600-495a-11eb-94be-6173bbd5553a.jpg)
![02](https://user-images.githubusercontent.com/23582534/103243470-40530900-495a-11eb-9635-2eed1c1c110a.jpg)
![04](https://user-images.githubusercontent.com/23582534/103243472-41843600-495a-11eb-8072-e5487be43cc4.jpg)

**Comment by barbudor** [comment]
Nice build @MrVulgaire  👍 

Did you salvaged the mounting plate + cover from an old wall device ? 
Or did you made it yourself ?

Thanks for sharing

**Comment by MrVulgaire** [comment]
Merci @barbudor 

I buy the box and the Cover :
https://www.hornbach.ch/shop/Betondose-1x1-mit-Deckel/6114134/artikel.html
https://www.hornbach.ch/shop/Hager-Blindabdeckung-mit-2-Eckloechern-weiss/5051686/artikel.html

I just made a few holes that are not well aligned :(

**Comment by barbudor** [comment]
Thanks @MrVulgaire
